sumologic-app-utils/src/sumoresource.py

```diff
@@ -631,27 +650,22 @@ def create_by_install_api(self, appid, appname, source_params, folder_name, *arg
             return {"APP_FOLDER_NAME": content["name"]}, app_folder_id
         else:
             print("%s installation failed." % appname)
-            response.raise_for_status()
+            raise Exception(response.text)
```

Comment:
response for status will raise specific error like Connection, Timeout etc any particular reason for creating new Exception?

Reply:
Yes, did not exactly remember but the error was not looking correct so had to do it